gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -3417,6 +3579,10 @@ subroutine Run ( gc, import, export, clock, rc )
       if (list(n)%disabled .or. ESMF_AlarmIsRinging(list(n)%end_alarm) ) then
          list(n)%disabled = .true.
          Writing(n) = .false.
+         if (MAPL_am_I_Root()) THEN
+            WRITE(*, 1999) n
+         ENDIF
+1999     FORMAT('Not Writing alarm ', i3, ' because end_alarm is ringing')
```

Comment:
Prefer to avoid FORMAT statements:
```suggestion
         if (MAPL_am_I_Root()) THEN
            WRITE(*, '("Not Writing arlarm ", i3, " because end alarm is ringing.") n
         ENDIF
```